test/test_file_blob.rb

```diff
@@ -522,6 +522,9 @@ def test_vendored
     assert sample_blob("myapp/My Template.xctemplate/___FILEBASENAME___.h").vendored?
     assert sample_blob("myapp/My Images.xcassets/some/stuff.imageset/Contents.json").vendored?
     assert !sample_blob("myapp/MyData.json").vendored?
+
+    # Jenkins
+    assert sample_blob("Jenkinsfile").vendored?
```

Comment:
By the way `lib/linguist/vendor.yml` suggests adding test coverage to `test/test_blob.rb#test_vendored` but it seems that almost all of the existing tests are here instead, so I tried to follow suit.